srtcore/core.cpp

```diff
@@ -2688,24 +2764,28 @@ bool CUDT::interpretSrtHandshake(const CHandShake& hs,
     }
 
     if (hs.m_iVersion < HS_VERSION_SRT1)
-        return true; // do nothing
+        return false; // do nothing
```

Comment:
That's wrong. I didn't mean to change it. Although I planned to return true or false here depending on the URQ code.